app/presenters/keywords_collection_presenter.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class KeywordsCollectionPresenter
```

Comment:
Related to this comment https://github.com/malparty/google-search-ruby/pull/46#discussion_r660288299, how about do the same also for this class?

```ruby
class KeywordsCollectionPresenter
  def initialize(keywords)
    @keywords = keywords
  end

  def groups
    keywords.group_by { |keyword| keyword.name[0].upcase.to_sym }
  end
  
  private
  
  attr_reader :keywords
end
```

Reply:
Makes perfect sense! :)